lib/actions.js

```diff
@@ -35,30 +35,39 @@ var ACCOUNT_STATUS_ACTION = {
 
 // Actions that send an email, and hence might make
 // us look like spammers if abused.
-var EMAIL_SENDING_ACTION = {
+const EMAIL_SENDING_ACTION = {
   accountCreate: true,
   recoveryEmailResendCode: true,
   passwordForgotSendCode: true,
   passwordForgotResendCode: true,
   sendUnblockCode: true
 }
 
+// Actions that can send sms, and could make us
+// very annoying to a user if abused.
+const SMS_SENDING_ACTION = {
+  connectDeviceSms: true
+}
+
 module.exports = {
 
-  isPasswordCheckingAction: function isPasswordCheckingAction(action) {
```

Comment:
this was here because it is useful during debugging / testing, but we can live without it I hope